### PR TITLE
Style safe autocorrect 2

### PR DESCRIPTION
**Pull request by JoeCohen**
Fixes the remaining safely autocorrectable Style offenses.
(Reduces net offenses by > 2%.)

### CODE REVIEW

**Comment by JoeCohen** [comment]
Converted to draft so that I can autocorrect a few more cops

File: app/models/glossary_term.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class GlossaryTerm < AbstractModel
              inverse_of: :best_glossary_terms)
   belongs_to :user
   belongs_to :rss_log
-  has_and_belongs_to_many :images, -> { order "vote_cache DESC" }
+  has_and_belongs_to_many :images, -> { order("vote_cache DESC") }
```
Comment (nimmolo) [inline app/models/glossary_term.rb:14]: Not that it matters, but maybe `order(vote_cache: :desc)`?

**Comment by nimmolo** [comment]
My main takeaway from this is, I've got to fix my autocorrect! :)
Seems good to me, apart from the one insignificant tweak.

**Comment by pellaea** [comment]
This looks very familiar.  I could swear I'd already signed off on this one!  Still looks perfectly good to me.

**Pull request merged**